pom.xml

```diff
@@ -47,7 +47,7 @@
         <pdfgen-core.version>1.1.50</pdfgen-core.version>
         <confluent.platform.version>7.9.1</confluent.platform.version>
         <mockito.version>5.17.0</mockito.version>
-        <ung-deltakelse-opplyser.version>1.9.0</ung-deltakelse-opplyser.version>
+        <ung-deltakelse-opplyser.version>2.0.0</ung-deltakelse-opplyser.version>
```

Comment:
2.0.1 er siste versjon med fiks for riktig oppgavetypedata for endret start og sluttdato.